app/src/main/res/drawable-v21/c24_normal_row_ripple.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
+    android:color="?android:colorControlHighlight">
+    <item android:drawable="@drawable/c24_normal_row_background"/>
+</ripple>
```

Comment:
Just as info: Ripple(s) are only working on Android Lollipop (Api version 21+). This is the touch animation feedback introduced with material design. Therefore when applied as drawable from code, a check for isLollipop version is required. When applied in style, it is sufficient to apply it on styles-v21 folder, which are the styles override for different versions in Android. (See c24_button_blue styles). https://developer.android.com/reference/android/graphics/drawable/RippleDrawable.html